scripts/cesm3_LTso_workflow_TEMPLATE.sh

```diff
@@ -0,0 +1,338 @@
+'PM25_SRF', 'pom_a1', 'pom_a4', 'PR', 'PRECC', 'PRECT', 'PS', 'PSIRS', 'PSL', 'PTTEND', 'Q', 'QFLX', 'QFLX_EVAP_TOT', 
+'QFLX_SNOW_DRAIN', 'QREFHT', 'QRL', 'QRS', 'QSNOFRZ', 'QSNOW', 'RAIN', 'RAIN_ICE', 'RELHUM', 'RHREFHT', 'SHFLX', 
+'SNOCAN', 'SNOW', 'SO2', 'so4_a1', 'so4_a2', 'so4_a3', 'soa1_a1', 'soa1_a2', 'soa2_a1', 'soa2_a2', 'soa3_a1', 
+'soa3_a2', 'soa4_a1', 'soa4_a2', 'soa5_a1', 'soa5_a2', 'SOLIN', 'SOLLD', 'SOLSD', 'T', 'T700', 'T850', 'TAUBLJX', 
+'TAUBLJY', 'TAUGWX', 'TAUGWY', 'TAUX', 'TAUY', 'TGCLDIWP', 'TGCLDLWP', 'THzm', 'TMO3', 'TMQ', 'TOT_CLD_VISTAU', 
+'TREFHT', 'TREFHTMN', 'TREFHTMX', 'TREFMNAV', 'TREFMXAV', 'TS', 'U', 'U10', 'UTGWORO', 'UTGWSPEC', 'UVzm', 'UWzm', 
+'Uzm', 'V', 'VTHzm', 'Vzm', 'WD_NH3', 'WD_NH4', 'WD_NOY', 'Wzm', 'Z1000', 'Z3', 'Z500', 'ZMFLXSNW'
+
+EOF
+
+fi
+
+if [[ $do_hr6 == true ]]; then
+cat <<EOF>> user_nl_cam
+
+! 6 hourly
+fincl3 = 'FLDS', 'FSDS', 'PRECT', 'PSL', 'PS', 'QREFHT', 'RAIN', 'SNOW', 'RHREFHT', 
+'TMQ', 'TREFHT', 'TS', 'U', 'U10', 'V', 'Z1000', 'Z3', 'Z500', 'Z700', 'Z925', 
+'PRECTMX', 'RELHUM100', 'RELHUM500', 'RELHUM850', 'Q', 'UREFHT', 'VREFHT'
+
+EOF
+
+fi
+
+if [[ $do_hr3 == true ]]; then
+cat <<EOF>> user_nl_cam
+
+! 3 hourly
+fincl4 = 'QREFHT', 'PRECT', 'SNOW', 'PS', 'FLDS', 'FSDS', 'TREFHT', 'RHREFHT', 
+'num_a1', 'num_a2', 'num_a3', 'num_a4', 'bc_a1', 'bc_a4', 'dst_a1', 'dst_a2', 'dst_a3', 
+'NH4', 'pom_a1', 'pom_a4', 'soa1_a1', 'soa1_a2', 'soa2_a1', 'soa2_a2', 'soa3_a1', 
+'soa3_a2', 'soa4_a1', 'soa4_a2', 'soa5_a1', 'soa5_a2', 'so4_a1', 'so4_a2', 'so4_a3', 
+'ncl_a1', 'ncl_a2', 'ncl_a3', 'ncl_c1', 'ncl_c2', 'ncl_c3', 'SO2', 'U10'
+
+EOF
+
+fi
+
+########################
+## user_nl_clm
+########################
+
+cat <<EOF> user_nl_clm
+reseed_dead_plants = .true.
+EOF
+
+cat <<EOF> user_nl_cpl
+histaux_l2x1yrg = .true.
+EOF
+
+cat <<EOF> user_nl_cice
+EOF
+
+cat <<EOF> user_nl_mom
+EOF
+
+
+mkdir -p $SCRATCH/${CASENAME}/run/INPUT
+cd $SCRATCH/${CASENAME}/run/INPUT
+ln -s /glade/derecho/scratch/gmarques/for_cecile/198/INPUT/* .
+
+cd $CASEROOT
+./preview_namelists
+
+
+########################
+## timeseries
+########################
+if [[ $do_tseries == true ]]; then
+   ## => the timeseries will be part of pf the workflow 
+   echo "do_tseries == $do_tseries"
+fi
+
+########################
+## CUPID
+########################
+if [[ $do_cupid == true ]]; then
+   ## => Cupid
+   echo "do_cupid == $do_cupid"
+fi
+
+
+########################
+## CMORization
+########################
+if [[ $do_cmor == true ]]; then
+   ## => CMORization
+   echo "do_cmor == $do_cmor"
+fi
+
+## Right now Jim needs to separate all the time frequency (monthly, daily, etc) in different directories. 
+
+########################
+## build and submit
+########################
+
+cd ${CASEROOT}
+./xmlchange JOB_PRIORITY=premium
+./xmlchange PROJECT=${project},RESUBMIT=2,STOP_N=4,STOP_OPTION=nyears
```

Comment:
PROJECT would already be set in the create_newcase call so you don't really need to change it here. You could also set the wallclock time that's done below in the create_newcase line as well with the "--walltime" option there. But, I'll also point out that the default walltime is 12 hours already so it doesn't really need to be overridden.

Reply:
I think that there will be options to set 
./xmlchange JOB_PRIORITY=premium 
and possibly
./xmlchange JOB_QUEUE=special --force --subgroup case.run
and as Erik points out we don't need to set JOB_WALLCLOCK_TIME here, it's already done in create_newcase.